modules/navier_stokes/src/fvkernels/WCNSFV2PMomentumDriftFlux.C

```diff
@@ -26,6 +26,7 @@ WCNSFV2PMomentumDriftFlux::validParams()
   params.addParam<MooseFunctorName>("w_slip", "The slip velocity in the z direction.");
   params.addRequiredParam<MooseFunctorName>("rho_d", "Dispersed phase density.");
   params.addParam<MooseFunctorName>("fd", 0.0, "Fraction dispersed phase.");
+  params.set<unsigned short>("ghost_layers") = 5;
```

Comment:
I think with the latest modifications you won't need this line.

Reply:
good catch!